ci/travis/install-dependencies.sh

```diff
@@ -94,6 +94,15 @@ install_miniconda() {
         conda="${miniconda_dir}/bin/conda"
         ;;
     esac
+  else
+    case "${OSTYPE}" in
+      darwin*)
+        # This is the default Miniconda directory on Mac
```

Comment:
and on GHA? It seems like GHA has miniconda preinstalled?

Reply:
Yeah. Or at least it has the `conda` command preinstalled, and this is the directory that it writes to.